guide/how-it-works/payment-request-formats.md

```diff
@@ -9,129 +9,253 @@ redirect_from:
 
-### Lightning invoices ([BOLT 11](https://github.com/lightning/bolts/blob/master/11-payment-encoding.md))
+## Lightning payment request formats
 
-Lightning invoices are the basic payment mechanism on the Lightning network. They are usually set to expire after 1 hour and should only be paid once for best security and privacy. Invoices must be created by the recipient and shared with the sender, who then makes the payment.
+### Invoice
+
+<div class="center" markdown="1">
+
+{% include image.html
+   image = "/assets/images/guide/how-it-works/payment-request-formats/invoice.jpg"
+   retina = "/assets/images/guide/how-it-works/payment-request-formats/invoice@2x.jpg"
+   alt-text = "A QR code of an invoice"
+   width = 400
+   height = 500
+   layout = "float-right-desktop"
+%}
+
+An invoice is the basis of payment requests on lightning. Most other lightning payment request formats build on top of invoices. How they work on a technical level is defined in [BOLT 11](https://github.com/lightning/bolts/blob/master/11-payment-encoding.md).
+
+Invoices are single use payment requests which have built-in expiries, set to 60 minutes by default. Invoice expiration times can be configured for different use cases such as [Hold invoices](https://bitcoinops.org/en/topics/hold-invoices/). These have long expiration times that allow a receiver to accept the payment at a later time.
+
+Invoices can also contain other pieces of meta data useful for users, like a description detailing what the invoice is for. A name can be included in the description via [NameDesc](https://github.com/lightning/blips/pull/11), which lets the sender know who they are paying.
+
+Requesters' node public keys are included in their generated invoices, so they should not be shared publicly to preserve privacy.
+
+</div>
 
 {% include tip/open.html color="red" icon="forbid" label="Lightning invoices are not addresses" %}
 
-It could be tempting to refer to a Lightning invoice as a “lightning address.” Doing so could be confused with the [lightning address]({{'/guide/how-it-works/payment-request-formats/#lightning-addresses' | relative_url}}) protocol described below. Therefore, avoid calling an invoice an address. A “payment request” would be a better synonym for “invoice.”
+It could be tempting to refer to a lightning invoice as a “lightning address.” Doing so could be confused with the [lightning address]({{'/guide/how-it-works/payment-request-formats/#lightning-addresses' | relative_url}}) protocol described below. Therefore, avoid calling an invoice an address. A “payment request” would be a better synonym for “invoice.”
```

Comment:
```suggestion
It could be tempting to refer to a lightning invoice as a “lightning address.” Doing so could be confused with the [lightning address]({{'/guide/how-it-works/payment-request-formats/#lightning-addresses' | relative_url}}) protocol described below. Therefore, avoid calling an invoice an address. “Payment request” is a better synonym for “invoice.”
```